exe/Wrapper.hs

```diff
@@ -79,6 +80,10 @@ main = do
           putStrLn hlsVer
           putStrLn "Tool versions found on the $PATH"
           putStrLn $ showProgramVersionOfInterest programsOfInterest
+          putStrLn "Tool versions found by cradle"
```

Comment:
```suggestion
          putStrLn "Tool versions in your project"
```

No average user knows what a cradle is, maybe we can avoid throwing vocabulary at users in this case.